src/pages/Player.vue

```diff
@@ -341,6 +384,44 @@ export default {
   },
   computed: {
     ...mapGetters("auth", ["isAuthenticated"]),
+    scorecardProgress() {
+      // progress value (0-100) to be passed to the Scorecard component
+      const totalAttempted = this.numOfCorrect + this.numOfWrong;
+      if (totalAttempted == 0) return 0;
+      return (this.numOfCorrect / (this.numOfCorrect + this.numOfWrong)) * 100;
```

Comment:
```suggestion
      if (!totalAttempted) return 0;
      return (this.numOfCorrect / totalAttempted) * 100;
```

Reply:
Here, I can use `totalAttempted` instead of `(this.numOfCorrect + this.numOfWrong)` as I've already calculated it above.

But I don't think writing `totalAttempted == 0` as `!totalAttempted` is ideally correct. `totalAttempted` is a number data type. Why `!totalAttempted` works is because of coercion. This was in chapter 2 of YDKJS